src/selector/isChildOf.js

```diff
@@ -0,0 +1,13 @@
+(function(factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['$'], factory);
+    } else {
+        factory(window.Zepto || window.jQuery);
+    }
+}(function($) {
+    $.extend($.fn, {
+        isChildOf: function(selector) {
+            return (selector) ? $(this).parents().filter(selector).length > 0 : false;
```

Comment:
So I think you don't need to check the existence of selector. If you're worried about a default, you can do this:

```
return !!$(this).parents().filter(selector || '').length;
```